helper/schema/provider.go

```diff
@@ -60,11 +60,20 @@ type Provider struct {
 	// provider doesn't need to be configured, this can be omitted.
 	ConfigureContextFunc ConfigureContextFunc
 
+	// StopFunc is called from the gRPC server Stop call. This does not
```

Comment:
Only nitpick is the wording here, I find the phrase `Terraform graph` a bit ambiguous, I would suggest something like
```
StopFunc is called from the gRPC server Stop call. This is an opportunity to cleanup things outside of Terraform's resource lifecycle and management (ie. session files, etc).
```
I'm no wordsmith though, what is here is already pretty good.

Reply:
Want to write this up as a suggestion and I can merge it?